pkg/configs/db/db.go

```diff
@@ -77,7 +78,8 @@ func New(cfg Config) (DB, error) {
 		if err != nil {
 			return nil, fmt.Errorf("Could not read database password file: %v", err)
 		}
-		u.User = url.UserPassword(u.User.Username(), string(passwordBytes))
+		password := strings.TrimSpace(string(passwordBytes))
+		u.User = url.UserPassword(u.User.Username(), password)
```

Comment:
Looks good, but it's not complete. Ideally for every change we modify or add a tests.

you can extract a function that returns url `u` and error as needed and test that function.

Reply:
@friedrichg apologies for the late response, I can't seem to find a test file for this file. Should I create a new one or am I missing something?